<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRoomStatus</class>
 <widget class="QMainWindow" name="EditRoomStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="Reservable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>421</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Room">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Room </string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>210</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Room Status</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_RoomStatus"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Room Number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_RoomNo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Room Type    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_RoomType"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Vehicle">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Vehicle</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>210</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Vehicle Status</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_VehicleStatus"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Vehicle Number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_VehicleNo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Vehicle Type    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_VehicleType"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>240</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="4">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Available Single Room  </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Available Double Room</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Available Limousine Car</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Available Royal Suite  </string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Available Cars</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Available Busses</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="LogOut">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Update">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>421</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="editRoom">
       <property name="text">
        <string>Edit Room Status</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editVehicle">
       <property name="text">
        <string>Edit Vehicle Status</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
